_layouts/updates.html

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
Does this page really need to be its own layout? Would an `{% include %}` work?
The latter feels like it's better to me as it would greatly reduce the duplication of the page layout code.